### ISSUE TITLE
[BUG] Conditional classes should not trigger conflicting classnames error

### ISSUE THREAD

**Issue by francoismassart** [francoismassart/eslint-plugin-tailwindcss#21]
```
  ctl(`
    text-white
    rounded-md
    py-5
    px-10
    text-sm
    ${
      type === 'primary' &&
      `
        bg-black
        hover:bg-blue
      `
    }
    ${
      type === 'secondary' &&
      `
        bg-transparent
        hover:bg-green
      `
    }
    disabled:cursor-not-allowed
  `)
```

Linting this code reports an error for contraddicting classes for `bg-black` and `bg-green` when it should not because it is either one or the other...
